src/py/flwr/client/clientapp/app.py

```diff
@@ -196,11 +197,11 @@ def get_token(stub: grpc.Channel) -> Optional[int]:
         return None
 
 
-def pull_message(
+def pull_clientappinputs(
     stub: grpc.Channel, token: int
 ) -> tuple[Message, Context, Run, Optional[Fab]]:
-    """Pull message from SuperNode to ClientApp."""
-    log(INFO, "Pulling ClientAppInputs for token %s", token)
+    """Pull ClientAppInputs from SuperNode."""
+    log(INFO, "[flwr-clientapp] Pull ClientAppInputs for token %s", token)
```

Comment:
Just to be consistent.
```suggestion
    log(INFO, "[flwr-clientapp] Pulls ClientAppInputs for token %s", token)
```